OTPKit/Features/MapExtension/MapMarkingView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MapMarkingView.swift
+//  OTPKit
+//
+//  Created by Hilmy Veradin on 18/07/24.
+//
+
+import SwiftUI
+
+public struct MapMarkingView: View {
+    @ObservedObject private var locationManagerService = LocationManagerService.shared
+
+    public init() {}
+    public var body: some View {
+        VStack {
+            Spacer()
+
+            VStack(spacing: 16) {
+                Button(action: {
+                    locationManagerService.toggleMapMarkingMode(false)
+                    locationManagerService.selectCoordinate()
+                }, label: {
+                    Text("Add Map Location")
+                })
+                .padding(.all)
+                .background(Color.gray)
```

Comment:
Hi @aaronbrethorst, I wonder do you have any suggestion on this color? Seems like the contrast is not good for our texts...